daemon/logger/jsonfilelog/jsonfilelog.go

```diff
@@ -61,6 +61,15 @@ func New(info logger.Info) (logger.Logger, error) {
 		}
 	}
 
+	var compress = true
+	if compressString, ok := info.Config["compression"]; ok {
```

Comment:
Do we need to validate that this is only set when log rotation is enabled?

Reply:
OK, will do.